streams/src/test/java/org/apache/kafka/streams/processor/internals/ProcessorStateManagerTest.java

```diff
@@ -117,6 +120,11 @@
     private OffsetCheckpoint checkpoint;
     private StateDirectory stateDirectory;
 
+    private Position persistentPosition;
+    private Position nonPersistentPosition;
+    private StateStorePositionCheckpoint persistentCheckpoint;
+    private StateStorePositionCheckpoint nonPersistentCheckpoint;
```

Comment:
I think this might be a relic... Non-Persistent stores won't write a checkpoint, right?